docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -341,183 +341,34 @@ Now that you have set up your robot and are able to control your motor, you can
+### Configure the ADC sensor as a module from the registry
 
 _Resources_ refer to the different [components](/components/) and [services](/services/) Viam provides for robots to use.
 _Components_ refer to types of hardware, and each component's built-in `models` support the most common models of this hardware.
 For example, the [sensor component](/components/sensor/) has an `ultrasonic` model built in for the ubiquitous [ultrasonic sensor](https://www.sparkfun.com/products/15569).
 
-However, there are many different types of sensors used for sensing different things across the [Internet of Things](https://medium.com/@siddharth.parakh/the-complete-list-of-types-of-sensors-used-in-iot-63b4003ab6b3).
-Although the capacitive soil moisture sensor is not currently one of Viam's built-in models, you can use the Viam Python SDK to configure this sensor as a [custom resource](/extend/custom-components-remotes/) implementing the Viam [sensor class](https://python.viam.dev/autoapi/viam/components/sensor/sensor/index.html), making it a model of sensor available for you to use on your robot.
+However, there are many different types of sensors used for sensing different things across the [Internet of Things](https://medium.com/@siddharth.parakh/the-complete-list-of-types-of-sensors-used-in-iot-63b4003ab6b3). Although the capacitive soil moisture sensor is not currently one of Viam's built-in models, you can add it as a module.
 
-Create your custom sensor resource in 3 steps:
+A _module_ provides one or more modular resources, which add resource types ([components](/components/) and [services](/services/)) or models that are not built into Viam. A module can be added to your robot from the Viam registry.
 
-1. Code a `MoistureSensor` class implementing the `GetReadings()` method that belongs to all members of the Viam sensor class, and instantiate this class on a RPC server in the `main()` function of your code.
-2. Add this "sensor server" as a [remote part](/manage/parts-and-remotes/) of your `plant-watering-robot`.
-3. Add a command that runs the program you coded instantiating the "sensor server" as a _process_ of your robot.
+The Viam registry (/extend/modular-resources/) allows hardware and software engineers to collaborate on their robotics projects by writing and sharing custom modules with each other. You can add a module from the Viam registry directly from your robot’s Configuration tab in the Viam app, using the + Create component button.
 
-#### Code the `MoistureSensor` Class
+Add the mcp300x-adc-sensor module to your robot in 3 steps:
 
-If you haven't already, install the Viam Python SDK by following the instructions [on GitHub](https://python.viam.dev/).
+1. Go to your robot and from the create component section, search mcp300x.
+2. Add this module to your robot.
+3. Add the necessary attributes. For example, if you have a moisture sensor on channel 0, and your sensor_pin is 8, your configuration should look like this:
```

Comment:
```suggestion
3. Add the necessary attributes.
For example, if you have a moisture sensor on channel 0, and your sensor_pin is 8, your configuration should look like this:
```
Not a major change at all we just try to have every sentence put on a newline in docs

Reply:
(and indented to match)
```suggestion
3. Add the necessary attributes. 
    For example, if you have a moisture sensor on channel 0, and your sensor_pin is 8, your configuration should look like this:
```